apm-server/README.md

```diff
@@ -40,17 +41,26 @@ See [supported configurations][] for more details.
 
 ## Installing
 
-This chart is tested with the latest 7.9.2 version.
+This chart is tested with the latest 7.9.3-SNAPSHOT version.
 
 ### Install released version using Helm repository
 
 * Add the Elastic Helm charts repo:
 `helm repo add elastic https://helm.elastic.co`
 
 * Install it:
-  - with Helm 2: `helm install --name apm-server --version 7.9.2 elastic/apm-server`
-  - with [Helm 3 (beta)][]: `helm install apm-server --version 7.9.2 elastic/apm-server`
+  - with Helm 2: `helm install --name apm-server --version <version> elastic/apm-server`
+  - with [Helm 3 (beta)][]: `helm install apm-server --version <version> elastic/apm-server`
+
+### Install development version from a branch
+
+* Clone the git repo: `git clone git@github.com:elastic/helm-charts.git`
 
+* Checkout the branch : `git checkout 7.9`
+
+* Install it:
+  - with Helm 2: `helm install --name apm-server ./helm-charts/apm-server`
+  - with [Helm 3 (beta)][]: `helm install apm-server ./helm-charts/apm-server`
```

Comment:
What is the point of removing the `7.9.3-SNAPSHOT` from this section.
Should we expect not removing it from the readme when doing releases?

Reply:
Oh yes, it seems it was added in the previous pr but not in the doc 👍